app/controllers/requests_controller.rb

```diff
@@ -23,8 +23,9 @@ def edit
   # POST /requests
   def create
     @request = Request.new(request_params)
-
     if @request.save
+      Mailer.send_generic_email(false, @request.email, 'workshop.portal@gmail.com', I18n.t('requests.email.topic'),
```

Comment:
@bjrne Take a look at how the "unreasonably long event time span" is stored: https://github.com/hpi-swt2/workshop-portal/blob/7d76af4af777d74b8917659ad7e9d6d3db58ab83/config/application.rb#L30-L30